tests/test_flood_stage.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+
+from hydrofunctions.flood_stage import get_flood_stage
+
+
+class TestFloodStage(unittest.TestCase):
```

Comment:
This is great that you included a test! I love it!

Since your new function is going to make a request out to the internet, testing it this way is going to also make a request. Unfortunately, that will take a little extra time, and the test could fail if the service goes down, even though the code was correct!

To fix this, you could use mocking.  In this case, you would create a 'mock' version of the Requests class, and then you would test to see whether your function called it correctly.

Reply:
Actually it is testing the function which receives all stages (so no request here) and filters those the user wants. But mock testing is even better :)